### ISSUE TITLE
user docs HTML: intro.txt

### ISSUE THREAD

**Issue by priceld** [neovim/neovim#30937]
`gen_help_html.lua` problem at: https://neovim.io/doc/user/intro.html

Context:

Several of the ASCII drawings as well as the "from mode to mode" table are malformed when viewing on the web.

E.g.
![Image](https://github.com/user-attachments/assets/3a757bb7-4d72-4573-9a83-e05e1468dce0)
![Image](https://github.com/user-attachments/assets/44c6f225-4625-42a1-84ad-4651cb5c1479)

**Comment by clason** [comment]
Yes, that's a known issue. Tabular/drawing material is not really something that vimdoc exposes in a structured manner.

**Comment by justinmk** [comment]
In this case it's not about the tables, but rather `fix_tab_after_conceal` isn't doing its magic quite right (or maybe it wasn't called for these lines, or some other tab-char rendering issue). https://github.com/neovim/neovim/blob/86832dcd1922eee37acc26310f72aa70def1514b/scripts/gen_help_html.lua#L465

**Comment by zeertzjq** [comment]
These are codeblocks. No concealing happens in these lines.

**Comment by boltlessengineer** [comment]
It seems to be fixed now. Does this issue still need some work?
